repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -2,15 +2,104 @@
 
 include::../_includes/in-progress.adoc[]
 
-<<add-header.adoc#,Add Header Filter>>::
-    Enables users to add new headers and header values to a request and/or response.
+== <<add-header.adoc#,Add Header Filter>>
+Enables users to add new headers and header values to a request and/or response.
 
-<<cors.adoc#,CORS Filter>>::
-    Enables users to add support for Cross-Origin Resource Sharing requests.
+== <<api-validator.adoc#,API Validator Filter>>
+*IN PROGRESS*
 
-<<rackspace-auth-user.adoc#,Rackspace Auth User Filter>>::
-    For identity usage for rate limiting and user access events.
+== <<body-extractor-to-header.adoc#,Body Extractor to Header Filter>>
+*IN PROGRESS*
 
-<<scripting.adoc#,Scripting Filter>>::
-    Enables users to write custom filters for Repose using a
-    variety of scripting languages.
+== <<body-patcher.adoc#,Body Patcher Filter>>
+*IN PROGRESS*
+
+== <<compression.adoc#,Compression Filter>>
+*IN PROGRESS*
+
+== <<content-type-stripper.adoc#,Content Type Stripper Filter>>
+*IN PROGRESS*
+
+== <<cors.adoc#,CORS Filter>>
+Enables users to add support for Cross-Origin Resource Sharing requests.
+
+== <<derp.adoc#,Delegation Response Processor (DeRP) Filter>>
+*IN PROGRESS*
+
+== <<destination-router.adoc#,Destination Router Filter>>
+*IN PROGRESS*
+
+== <<echo.adoc#,Echo Filter>>
+*IN PROGRESS*
+
+== <<exception.adoc#,Exception Filter>>
+*IN PROGRESS*
+
+== <<forwarded-proto.adoc#,Forwarded Protocol Filter>>
+*IN PROGRESS*
+
+== <<header-normalization.adoc#,Header Normalization filter>>
+*IN PROGRESS*
+
+== <<header-translation.adoc#,Header Translation Filter>>
+*IN PROGRESS*
+
+== <<header-user.adoc#,Header User Filter>>
+*IN PROGRESS*
+
+== <<herp.adoc#,Highly Efficient Record Processor (HERP) Filter>>
+*IN PROGRESS*
+
+== <<ip-user.adoc#,IP User Filter>>
+*IN PROGRESS*
+
+== <<iri-validator.adoc#,IRI Validator Filter>>
+*IN PROGRESS*
+
+== <<keystone-v2.adoc#,Keystone V2 Filter>>
+*IN PROGRESS*
+
+== <<keystone-v2-basic-auth.adoc#,Keystone v2 Basic Auth Filter>>
```

Comment:
I'm not sure if this goes here in the ordering.  If you order the filters leaving out the "Filter" part (which I'm totally cool with), then this is in the correct spot.  Otherwise, this would go above the "Keystone v2 Filter".  I think it should stay here, but I wanted to point that out.